### PR TITLE
ported to polymer 2.0

### PR DESCRIPTION
**Pull request by fincha**
fixed events
fixed resizing
fixed demo page
fixed npm/bower on ing standards
updated bower/package.json

### CODE REVIEW

File: .bowerrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "strict-ssl": false,
+    "registry" : "https://artifactory.ing.net/artifactory/api/bower/Bower",
+    "shorthand-resolver": "art://{{owner}}/{{package}}",
+    "resolvers" : [
+        "bower-ing-resolver",
+        "bower-art-resolver"
+    ],
+    "no-proxy": "ing.net"
+}
```
Comment (tlouisse) [inline .bowerrc]: This file and .npmrc should not be committed I think ;)

File: paper-calendar.html [hunk 2]
```diff
@@ -707,9 +715,15 @@
...
             _resizeHandler: function () {
+                /**
+                 * after upgrading to polymer 2.0 for unknown reason the offsetWidth is returning 0
+                 * was able to fix it only with setTimeout. Pray to be forgiven by the gods of javascript (c).
+                 */
+                setTimeout(function() {
                 this._containerWidth = this.$.calendar.offsetWidth;
                 this._positionMonths();
                 this._recalculateSelectionCircle();
+                }.bind(this), 1)
```
Comment (tlouisse) [inline paper-calendar.html]: Maybe it's neater to align this with Polymer.Base.afterNextRender ?

File: .bowerrc (hunk 1 above)
Comment (fincha) [inline .bowerrc]: done

File: paper-calendar.html (hunk 2 above)
Comment (fincha) [inline paper-calendar.html]: hm, is this polymer 2.0 function only?

**Comment by fincha** [comment]
fixed tests

File: paper-date-picker-dialog-style.html [hunk 3]
```diff
@@ -38,5 +17,27 @@
         };
       }
     </style>
+    <custom-style>
```
Comment (Frederik76) [inline paper-date-picker-dialog-style.html]: Why didn't you combine the two style sections?

File: paper-date-picker.html [hunk 4]
```diff
@@ -55,12 +55,14 @@
 @demo demo/index.html
 -->
 
-<style is="custom-style" include="paper-date-picker-dialog-style">
-  /* includes dialog style at document-level for backward compatibility */
-</style>
-
 <dom-module id="paper-date-picker">
   <template>
+    <custom-style>
```
Comment (Frederik76) [inline paper-date-picker.html]: Why didn't you combine both style sections?

**Pull request merged**